src/components/ux-orbit/ux-orbit.steps.js

```diff
@@ -15,4 +16,21 @@ module.exports = function () {
 			this.client.loadComponentWithUseCase(componentName, useCase, callback);
 	});
 
+	this.Then(/^I should see (\d+) slides$/, function (numElements, callback) {
```

Comment:
Can this be reworded to be generic? Give it a selector and return the count of elements found?